wger/exercises/views/exercises.py

```diff
@@ -77,7 +78,72 @@ def get_queryset(self):
         '''
         Filter to only active exercises in the configured languages
         '''
-        languages = load_item_languages(LanguageConfig.SHOW_ITEM_EXERCISES)
+    #     language = None
```

Comment:
This a huge chunk of commented out code. I'd advise you remove it all the same if it is no longer needed,  however be sure it does not affect any other work

Reply:
Thank you. Feedback implemented, please review.